core/_interfaces/test-fixture.i.ts

```diff
@@ -8,6 +8,7 @@ export interface ITestFixture {
 
   focussed: boolean;
   tests: Array<ITest>;
+  description: string;
```

Comment:
`description` is used for a test as well, so I think we should use the same for both for now, and open an issue to discuss changing both of them :)